dist/palindrom-dom.js

```diff
@@ -2130,7 +2130,7 @@ var Palindrom = (function() {
     this.onLocalChange = options.onLocalChange || noop;
     this.onRemoteChange = options.onRemoteChange || noop;
     this.onStateReset = options.onStateReset || options.callback || noop;
-
+    this.filter = options.filter || (operation => operation);
```

Comment:
I had the very same thought. `onBeforeLocalChange` conveys an event/callback => you don't need to return.

`filterLocalChange` is perfect IMO.